kafka-rest/src/main/java/io/confluent/kafkarest/DefaultKafkaRestContext.java

```diff
@@ -27,32 +33,35 @@
 public class DefaultKafkaRestContext implements KafkaRestContext {
 
   private final KafkaRestConfig config;
-  private ProducerPool producerPool;
   private KafkaConsumerManager kafkaConsumerManager;
+
   private Admin adminClient;
+  private Producer<byte[], byte[]> producer;
 
+  /**
+   * @deprecated Use {@link #DefaultKafkaRestContext(KafkaRestConfig)} instead.
+   */
+  @Deprecated
```

Comment:
why is this constructor still here?

Reply:
This is still used in confluent-security-plugins. I will have another PR there (after this one is merged) to remove these references, than I can yet another PR here to remove these deprecated elements.

Reply:
look forward to it :-)